tools/export/blender25/godot_export_manager.py

```diff
@@ -323,17 +348,19 @@ def execute(self,context):
                 object.select = False
             context.scene.objects.active = None
 
-            ### make particle duplicates, parent and select them
+            # Make particle duplicates, parent and select them
             nodes_to_be_added = []
             if group[self.idx].use_include_particle_duplicates:
-                for i,object in enumerate(group[self.idx].nodes):
+                for i, object in enumerate(group[self.idx].nodes):
                     if bpy.data.objects[object.name].type != "EMPTY":
-                        context.scene.objects.active = bpy.data.objects[object.name]
+                        context.scene.objects.active = bpy.data.objects[
+                            object.name]
```

Comment:
What's the limit for cropping lines in the tool you used? I'm not sure this change really adds to readability.